docs/consume-packages/Package-Source-Mapping.md

```diff
@@ -24,6 +24,10 @@ Older tooling will ignore the Package Source Mapping configuration. To use this
 
 Package Source Mappings will apply to all project types – including .NET Framework – as long as compatible tooling is used.
 
+## Related video

```

Comment:
I love that we're adding this. 

I'm wondering if we can tweak the title a little bit. Should it maybe be a video walkthrough or something like that?

Reply:
Sounds good! I didn't know what to put so I stole the heading on https://docs.microsoft.com/en-us/nuget/what-is-nuget. I'll do Video walkthrough.

Reply:
```suggestion
## Video walkthough
```

Reply:
![image](https://user-images.githubusercontent.com/94054/151865991-4680e2dc-7614-405e-b40d-178f225be7a0.png)
☠️